### ISSUE TITLE
409 Conflict error on "Confirmed" contacts

### ISSUE THREAD

**Issue by MegaphoneJon** [cividesk/com.cividesk.sync.constantcontact#2]
[I'm reporting this so it's documented for other potential users/yourself, but I'm not asking for this to be fixed!  If it was important enough, I'd do it myself or get a client to fund it.]

If a contact is marked as "Confirmed", the extension can't sync them.  The solution suggested [here](https://community.constantcontact.com/t5/Developer-Support-ask-questions/409-conflict-error-even-when-passing-in-unmodified-data/m-p/278796#M10058) is to use the Bulk API instead - but that's not a trivial undertaking.

I did some testing to see if we could just not pass the email address in on contact update - but that's a required value for the API.

**Comment by nganivet** [comment]
Yes, definitely not a trivial change. Specifically as this would be a 
trivial change on the CtCt side to correct this 'limitation' of theirs: 
just compare the newly supplied email with what they have on file, and 
if same do not do the update. Thanks for pointing this out Jon.

------ Original Message ------
